code/modules/modular_computers/file_system/programs/antagonist/contract_uplink.dm

```diff
@@ -150,6 +150,13 @@
 		data["contracts_completed"] = traitor_data.contractor_hub.contracts_completed
 		data["contract_rep"] = traitor_data.contractor_hub.contract_rep
 
+		data["CONTRACT_STATUS_INACTIVE"] = 1
+		data["CONTRACT_STATUS_ACTIVE"] = 2
+		data["CONTRACT_STATUS_BOUNTY_CONSOLE_ACTIVE"] = 3
+		data["CONTRACT_STATUS_EXTRACTING"] = 4
+		data["CONTRACT_STATUS_COMPLETE"] = 5
+		data["CONTRACT_STATUS_ABORTED"] = 6
+
```

Comment:
Can you not send this with data, because it's so much extra data per packet?

You could add these constants to javascript, like so:

```js
const CONTRACT_STATUS_INACTIVE = 1;
const CONTRACT_STATUS_ACTIVE = 2;
const CONTRACT_STATUS_BOUNTY_CONSOLE_ACTIVE = 3;
const CONTRACT_STATUS_EXTRACTING = 4;
const CONTRACT_STATUS_COMPLETE = 5;
const CONTRACT_STATUS_ABORTED = 6;
```

Reply:
Ah my bad still got the old systems habits